packages/ember-metal/lib/tags.js

```diff
@@ -39,36 +39,33 @@ export function tagFor(object, _meta) {
 export function markObjectAsDirty(meta, propertyKey) {
   let objectTag = meta.readableTag();
 
-  if (objectTag) {
+  if (objectTag !== undefined) {
     objectTag.dirty();
   }
 
   let tags = meta.readableTags();
-  let propertyTag = tags && tags[propertyKey];
+  let propertyTag = tags !== undefined ? tags[propertyKey] : undefined;
 
-  if (propertyTag) {
+  if (propertyTag !== undefined) {
     propertyTag.dirty();
   }
 
   if (propertyKey === 'content' && meta.isProxy()) {
     meta.getTag().contentDidChange();
   }
 
-  if (objectTag || propertyTag) {
+  if (objectTag !== undefined || propertyTag !== undefined) {
     ensureRunloop();
   }
 }
 
 let run;
-
-function K() {}
-
 function ensureRunloop() {
-  if (!run) {
+  if (run === undefined) {
     run = require('ember-metal').run;
   }
 
-  if (hasViews() && !run.backburner.currentInstance) {
-    run.schedule('actions', K);
+  if (hasViews()) {
+    run.backburner._ensureInstance();
```

Comment:
I had stupid idea .... what happens if `ensureRunloop` removed ? .... because `bb.schedule` anyway calls `_ensureRunloop` inside right so `ensureRunloop` function is necessary as tests running well without it :P